Makefile

```diff
@@ -635,13 +635,14 @@ helm-repo-add:
 istio-remote.yaml: $(HELM) $(HOME)/.helm helm-repo-add
 	cat install/kubernetes/namespace.yaml > install/kubernetes/$@
 	cat install/kubernetes/helm/istio-init/files/crd-* >> install/kubernetes/$@
-	$(HELM) dep update --skip-refresh install/kubernetes/helm/istio-remote
+	$(HELM) dep update --skip-refresh install/kubernetes/helm/istio
```

Comment:
In a separate PR we should move back the charts so dep update is not needed ( was added to allow sharing of charts with istio-remote)

Reply:
Yes @costinm, that's the plan